SparkPost/2web2ui
# FE-395:  Handle GET failures correctly in Subaccounts and Sending Domains

**How to test?**

- For subaccount page, open a link to a subaccount that does not exist (e.g. http://app.sparkpost.test/account/subaccounts/0).  The expected result is to be redirected to the subaccount list page with a warning.
- For sending domain page, open a link to a sending domain that does not exist (e.g. http://app.sparkpost.test/account/sending-domains/edit/0).  The expected result is to be redirected to the sending domains list page with a warning.
